packages/SwingSet/test/vo-test-harness/test-vo-test-harness.js

```diff
@@ -0,0 +1,52 @@
+import { test, runVOTest } from '../../tools/vo-test-harness.js';
+
+async function voTestTest(t, mode) {
+  let makeThing;
+
+  // eslint thinks `retainer` is unused, but eslint is very wrong
```

Comment:
Should we disable the rule? Typescript gets it right.

Reply:
The value is actually unused in the sense that it's never actually made use of, but the assignment pins the reference in memory for GC purposes which is why it's there.  How would TypeScript know that's what's going on here?